text/0026-rest-api/rest-api.yaml

```diff
@@ -2149,6 +2301,20 @@ components:
       required:
         - data
 
+    ReceiptsResponse:
+      properties:
+        data:
+          type: object
+          properties:
+            receipts:
```

Comment:
The list of items in the array is an object with { receiptPayload, milestoneIndex }

see https://github.com/gohornet/hornet/blob/550487688dbd42ce335165420b5e5f8b9ac5ca25/pkg/model/utxo/receipt.go#L15-L20